.gitignore

```diff
@@ -24,6 +24,27 @@
 .eplaunch
 testfiles/EPLaunchRun_*/
 
+# EnergyPlus output files in the testsilfes folder
```

Comment:
Just ignore a few EnergyPlus output files in the case of running inside the repo.